fixtures/manufacturers.json

```diff
@@ -73,7 +74,8 @@
   },
   "prolights": {
     "name": "Prolights",
-    "website": "http://prolights.it/"
+    "website": "http://prolights.it/",
+    "rdmId": 5584
```

Comment:
The ID is actually registered under "Music & Lights S.r.l." rather than one of their child brands Prolights...

Reply:
Hmm... For now that's okay, but we need to work out a method to reference each of their child brands in a fixture and don't have the RDM manufacturer ID unique test failing. It would be problematic to save the same RDM ID for different manufacturers (each child brand being one) since we couldn't easily link back from the `/rdm` page.

Reply:
Perhaps you need the concept of badge engineered/child brands, listed within the one manufacturer (and available from the manufacturer drop down). I don't know how you integrate that into adding a new fixture with a new child manufacturer though.

The /rdm page stuff shouldn't be too much of an issue then, as you could map based on the RDM model ID (as they'll be unique within the manufacturer) and even direct to a virtual child manufacturer page.